### DISCUSSION TITLE
Import Rererence Field to Firebase Using CSV

### DISCUSSION THREAD

**Discussion by polenvinagre** [AntlerVC/firetable#706] [Support / Q&A] [answered]
Hi! Could someone help me. How can I import a reference field to firebase?
I tried use json but it import a array of data.
I tried ref field but it was blank/empty.
I am using a csv file.

**Comment by notsidney** [comment, marked as answer]
Hey there, currently you can’t import Firestore references from CSV. I’ve created an issue for this: #707
